### PR TITLE
r: add page

### PR DESCRIPTION
**Pull request by YangVincent**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/r.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# R
```
Comment (sbrl) [inline pages/common/r.md]: It really is a single letter command that's uppercase? That's rather unusual.

File: pages/common/r.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+> R language interpreter.
+
+- Call a R interactive shell (REPL):
```
Comment (sbrl) [inline pages/common/r.md]: I'd prefer `an` instead of `a` in this instance, as `R` is a single character.

File: pages/common/r.md (hunk 1 above)
Comment (YangVincent) [reply pages/common/r.md]: Yup! Though I've seen many versions where it calls for the lower case, though that may just be aliased.

File: pages/common/r.md (hunk 2 above)
Comment (YangVincent) [reply pages/common/r.md]: Fixed!

File: pages/common/r.md (hunk 1 above)
Comment (agnivade) [reply pages/common/r.md]: I would really hate to break convention here. All our pages have the heading as lowercase. Unless there is a different command with a lower case `r`, I would recommend changing this to lowercase.

File: pages/common/r.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+- Call an R interactive shell (REPL):
+
+`R`
+
+- Check R version:
+
+`R --version`
+
+- Run file as input:
```
Comment (agnivade) [inline pages/common/r.md]: Change this to  - "Execute a file" to keep it consistent with pages of other programming languages.

File: pages/common/r.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+`R -f file.R`
```
Comment (agnivade) [inline pages/common/r.md]: Wrap file.R in tokens

File: pages/common/r.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# R
+
+> R language interpreter.
+
+- Call an R interactive shell (REPL):
```
Comment (agnivade) [inline pages/common/r.md]: Call => Start

**Pull request merged**